src/__tests__/__snapshots__/Select.test.tsx.snap

```diff
@@ -359,9 +359,12 @@ exports[`Select with multi selection and searchable should generate Select with
               placeholder="Select..."
               placeholderTextColor="#808080"
               style={
-                Object {
-                  "fontSize": 12,
-                }
+                Array [
+                  Object {
+                    "fontSize": 12,
+                  },
+                  undefined,
```

Comment:
właśnie też w TG się zastanawiam czy ok są te undefined w snapshotach (w sensie wiem, że to przez someProp && styles.somePropStyle, ale czy ogólnie może tak być)

Reply:
@koko57 
tak, to jest ok

tu przykładowy snapshot z repo react-native'a z undefined w tablicy ;) https://github.com/facebook/react-native/blob/main/Libraries/Components/ScrollView/__tests__/__snapshots__/ScrollView-test.js.snap#L55